notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -41,8 +41,7 @@ import kotlinx.coroutines.cancel
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 import java.text.SimpleDateFormat
-import java.util.Date
-import java.util.Locale
+import java.util.*
```

Comment:
don't

Reply:
you can thank android studio for that.

Reply:
change your code editor preferences
https://github.com/square/java-code-styles